sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb

```diff
@@ -0,0 +1,1041 @@
+    "ml_client.jobs.stream(name=job_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download the prediction result for metrics calculation\n",
+    "The output of forecast output is saved in JSON format. You can use it to calculate test set metrics and plot predictions and actuals over time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.download(job_name, download_path=\".\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fcst_df = pd.read_json(output_file, orient=\"table\")\n",
+    "fcst_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calculate the metrics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from metrics_helper import calculate_metrics\n",
+    "\n",
+    "calculate_metrics(fcst_df[target_column_name], fcst_df[\"predicted\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Forecast versus actuals plot.\n",
+    "We will join historical data with the predictions to plot predictions and actuals on a time series plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "history_data = pd.read_csv(\n",
+    "    \"./data/training-mltable-folder/bike-no-train.csv\",\n",
+    "    parse_dates=[time_column_name],\n",
+    ")\n",
+    "history_data.sort_values(by=time_column_name, inplace=True)\n",
+    "history_data = history_data.iloc[-4 * forecast_horizon :]\n",
+    "# Merge predictions to historic data.\n",
+    "df = pd.concat([history_data, fcst_df], sort=False, ignore_index=True)\n",
+    "df.set_index(time_column_name, inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Build the plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "plt.plot(df[[target_column_name, \"predicted\"]])\n",
+    "plt.xticks(rotation=45)\n",
+    "plt.title(f\"Predicted vs. Actuals\")\n",
+    "plt.legend([\"actual\", \"forecast\"])\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more details on what metrics are included and how they are calculated, please refer to [supported metrics](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-understand-automated-ml#regressionforecasting-metrics). You could also calculate residuals, like described [here](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-understand-automated-ml#residuals).\n",
```

Comment:
Please replace "You could also calculate residuals, like described ... " with "You could also calculate residuals as described ..."

Reply:
my bad, thanks for correcting